### ISSUE TITLE
[Bug] UnsupportedOperationException: Method getParent is not yet implemented in I18nReference.wrapPhraseInElement

### ISSUE THREAD

**Issue by gonzaloimprontus** [Atwix/phpstorm-plugin-feedback#85]
**Describe the bug** (*)

Please, describe as much as possible about of what the bug is. It will help to solve this issue faster.

**StackTrace from IDE**

```
Unhandled exception in [CoroutineName(DocumentationBrowser requests), StandaloneCoroutine{Cancelling}@544ce80e, Dispatchers.Default]

java.lang.UnsupportedOperationException: Method getParent is not yet implemented in com.atwix.magento.feature.i18n.reference.I18nReference$wrapPhraseInElement$1
	at com.atwix.magento.lang.MockPsiElement.getParent(MockPsiElement.kt:178)
	at com.intellij.psi.css.impl.util.CssUtil.isCssDeclaration(CssUtil.java:142)
	at com.intellij.psi.css.impl.util.CssDocumentationProvider.findDocumentationElement(CssDocumentationProvider.java:178)
	at com.intellij.psi.css.impl.util.CssDocumentationProvider.generateDoc(CssDocumentationProvider.java:227)
	at com.intellij.lang.documentation.CompositeDocumentationProvider.generateDoc(CompositeDocumentationProvider.java:135)
	at com.intellij.xml.util.documentation.HtmlDocumentationProvider.generateDocFromStyleOrScript(HtmlDocumentationProvider.java:157)
	at com.intellij.xml.util.documentation.HtmlDocumentationProvider.generateDoc(HtmlDocumentationProvider.java:77)
	at com.intellij.lang.documentation.CompositeDocumentationProvider.generateDoc(CompositeDocumentationProvider.java:135)
	at com.intellij.lang.documentation.psi.PsiElementDocumentationTarget.localDocHtml(PsiElementDocumentationTarget.kt:105)
	at com.intellij.lang.documentation.psi.PsiElementDocumentationTarget.localDoc(PsiElementDocumentationTarget.kt:90)
	at com.intellij.lang.documentation.psi.PsiElementDocumentationTarget.computeDocumentation(PsiElementDocumentationTarget.kt:77)
	at com.intellij.platform.backend.documentation.impl.ImplKt$computeDocumentation$2$documentationResult$1.invoke(impl.kt:29)
	at com.intellij.platform.backend.documentation.impl.ImplKt$computeDocumentation$2$documentationResult$1.invoke(impl.kt:28)
	at com.intellij.openapi.application.rw.InternalReadAction.insideReadAction(InternalReadAction.kt:108)
	at com.intellij.openapi.application.rw.InternalReadAction.access$insideReadAction(InternalReadAction.kt:15)
	at com.intellij.openapi.application.rw.InternalReadAction$tryReadCancellable$1.invoke(InternalReadAction.kt:92)
	at com.intellij.openapi.application.rw.InternalReadAction$tryReadCancellable$1.invoke(InternalReadAction.kt:91)
	at com.intellij.openapi.progress.CancellationKt.withCurrentJob$lambda$0(cancellation.kt:17)
	at com.intellij.openapi.progress.Cancellation.withCurrentJob(Cancellation.java:60)
	at com.intellij.openapi.progress.CancellationKt.withCurrentJob(cancellation.kt:17)
	at com.intellij.openapi.progress.CancellationKt.executeWithJobAndCompleteIt(cancellation.kt:126)
	at com.intellij.openapi.application.rw.CancellableReadActionKt.cancellableReadActionInternal$lambda$1$lambda$0(cancellableReadAction.kt:49)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1100)
	at com.intellij.openapi.application.rw.CancellableReadActionKt.cancellableReadActionInternal$lambda$1(cancellableReadAction.kt:47)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtilService.runActionAndCancelBeforeWrite(ProgressIndicatorUtilService.java:63)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:129)
	at com.intellij.openapi.application.rw.CancellableReadActionKt.cancellableReadActionInternal(cancellableReadAction.kt:45)
	at com.intellij.openapi.application.rw.InternalReadAction.tryReadCancellable(InternalReadAction.kt:91)
	at com.intellij.openapi.application.rw.InternalReadAction.access$tryReadCancellable(InternalReadAction.kt:15)
	at com.intellij.openapi.application.rw.InternalReadAction$tryReadAction$2.invoke(InternalReadAction.kt:77)
	at com.intellij.openapi.application.rw.InternalReadAction$tryReadAction$2.invoke(InternalReadAction.kt:72)
	at com.intellij.openapi.progress.CancellationKt.withCurrentJob$lambda$0(cancellation.kt:17)
	at com.intellij.openapi.progress.Cancellation.withCurrentJob(Cancellation.java:60)
	at com.intellij.openapi.progress.CancellationKt.withCurrentJob(cancellation.kt:17)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContext(coroutines.kt:193)
	at com.intellij.openapi.application.rw.InternalReadAction.tryReadAction(InternalReadAction.kt:72)
	at com.intellij.openapi.application.rw.InternalReadAction.readLoop(InternalReadAction.kt:64)
	at com.intellij.openapi.application.rw.InternalReadAction.access$readLoop(InternalReadAction.kt:15)
	at com.intellij.openapi.application.rw.InternalReadAction$runReadAction$4.invokeSuspend(InternalReadAction.kt:43)
	at com.intellij.openapi.application.rw.InternalReadAction$runReadAction$4.invoke(InternalReadAction.kt)
	at com.intellij.openapi.application.rw.InternalReadAction$runReadAction$4.invoke(InternalReadAction.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:169)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.application.rw.InternalReadAction.runReadAction(InternalReadAction.kt:39)
	at com.intellij.openapi.application.rw.PlatformReadActionSupport.executeReadAction(PlatformReadActionSupport.kt:29)
	at com.intellij.openapi.application.ReadActionSupport.executeReadAction$default(ReadActionSupport.kt:15)
	at com.intellij.openapi.application.CoroutinesKt.constrainedReadAction(coroutines.kt:55
```

**To Reproduce** (*)

**Screenshots**

**System information:**

 - Occurred: 2024/05/31
 - Plugin version: 2023.5.1/F
 - PhpStorm version: 2023.1.5 (PS-231.9414.10)
 - OS version: Linux 5.15.0-107-generic

**Comment by bohdan-harniuk** [comment]
Hello @gonzaloimprontus,

Thank you for taking the time to report this issue. We greatly appreciate your contribution to improving our project.

Our team is now reviewing the problem, and we'll work on resolving it as quickly as possible. We will keep you updated on our progress, and if we have any questions or need more information from you, we'll reach out.

Once again, thank you for your report, and we'll do our best to address this issue promptly.

Best regards,

**Comment by bohdan-harniuk** [comment]
Hello @gonzaloimprontus ,

Apologies for the delayed response.

The bug you reported has been resolved in plugin version 2023.5.2. Since then, there have been several updates, so we highly recommend upgrading to the latest version to take advantage of the newest features and enhancements.

If you encounter any further issues or have additional feedback, feel free to reach out.

Best regards,
